src/import/import.rs

```diff
@@ -386,6 +390,12 @@ impl<Client: ImportClient> ImportSSTJob<Client> {
             let file = self.sst.info.open()?;
             let upload = UploadStream::new(self.sst.meta.clone(), file);
             let store_id = peer.get_store_id();
+            while !self.client.is_store_available(store_id)? {
+                IMPORT_WAIT_STORE_AVAILABLE_COUNTER.inc();
```

Comment:
We need to reset the counter after the loop is broken.

Actually do we really need this metric since PD already records it 🤔

Reply:
It is the counter of how much times to wait for disk space available. See `src/import/client.rs`